src/entities/requests/BuildBuildingRequest.ts

```diff
@@ -0,0 +1,4 @@
+export default class BuildBuildingRequest {
```

Comment:
Why?

Reply:
Because they are classes without methods - would be an actual good use for a typescript interface type. Also if these stay this short it might be ok to put them into one file together.
Don't remember what else I ment with "remove".

Reply:
Using interfaces with no defines methods and only properties is somewhat against the definition of an interface, isn't it? Would feel wrong to use interfaces instead of classes here.

Reply:
In the sense you might know interfaces from other languages, maybe.
But javascript is able to create data objects from pure json like it is the case here. Typescript can use interfaces to type those data objects without the need to instantiate them like you would do with class based objects. IMHO this is actually what typescript interfaces are mostly ment for and obviously this is quite different to other languages like Java.